source/backend/api/Services/PropertyService.cs

```diff
@@ -281,6 +281,14 @@ public IList<PimsPropertyActivity> GetActivities(long propertyId)
             return _propertyActivityRepository.GetActivitiesByProperty(propertyId);
         }
 
+        public IList<PimsPropertyActivity> GetFileActivities(long managementFileId)
+        {
+            _logger.LogInformation("Getting property management activities for management file with id {managementFileId}", managementFileId);
+            _user.ThrowIfNotAuthorized(Permissions.ManagementView, Permissions.PropertyView);
```

Comment:
I believe in the past we learnt that using multiple permissions here will check for "any" so there is potential that a user who has a subset of the claims above would be able to call this method.

Please call `_user.ThrowIfNotAllAuthorized()` instead (or add permission checks in separate lines)

Reply:
I still see `_user.ThrowIfNotAuthorized()` with multiple permissions which only checks the user has any (not all) of the permissions - please change to `ThrowIfNotAllAuthorized()`

Reply:
wrong commit, sorry, up to date now.